tests/cases/standalone/common/alter/change_col_inverted_index.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE fox (
```

Comment:
Waiting for #5297 to be merged, you can check the effect here by checking the results of `SHOW INDEX FROM <table>`.

Reply:
added show index to the sqlness test